LanguageFeatures/Unquoted-imports/test.package1/dummy.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @description Library file for testing unquoted imports. Must be unavailable
+/// via `import 'package:...'` and its unquoted shortcut.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=unquoted-imports
+
+Never libraryId = "dummy.dart" as dynamic;
```

Comment:
How does it help in any test that this `libraryId` has type `Never`? If a particular declaration is unavailable then I'd expect every attempt to access it from another library to fail, and this would be just as true for an unsurprising declaration like `String libraryId = "Never used";`. Or is there something that makes a `libraryId` of type `Never` work better?

Reply:
Changed to a `String`.